docs/DeveloperGuide.md

```diff
@@ -138,10 +157,12 @@ This section describes some noteworthy details on how certain features are imple
 ### \[Completed\] Recording/ Updating Sales Data
 
 tCheck allows users to record and update the sales information on the drink sold. The command to use this feature is:
-`s-update A/NUM B/NUM ...` where:
+`s-update A/NUM [B/NUM] ...` where:
 * `A`, `B`, `C` are abbreviations for the drink types.
 * `NUM` refers to the number of drinks sold.
 
+The user may use this command for a single `Drink`, or multiple `Drink`s types, as indicated by the
```

Comment:
Maybe there are some missing words behind "by the"?

Reply:
Thanks for the catch! I will update this line